code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -85,6 +86,12 @@
 		stop_automated_movement = 1
 		walk_to(src,movement_target,0,3)
 
+/mob/living/simple_animal/cat/death()
+	.=..()
+	stat |= DEAD

```

Comment:
Aye, I am unable to find any bitfield operations conducted with mob.stat.

Machinery code does treat stat as a bitfield, but mob code treats it as an integer, with the possible values being 0, 1, and 2.